k8s/aat/common/rpe/bulk-scan-payment-processor.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bulk-scan-payment-processor
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bulk-scan-payment-processor
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan-payment-processor
+    version: 1.0.0
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
+      cpuRequests: '500m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "f666440f-43c0-4abb-9c20-2ce175a374f0"
+      image: hmctspublic.azurecr.io/bulk-scan/payment-processor:prod-77831e9a
+      environment:
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.5.0.jar
+      aadIdentityName: rpe
+      prometheus:
+        enabled: true
+    global:
```

Comment:
you don't need these 4 fields as you don't have keyvaults yet. Nothing wrong in having though.

Reply:
will keep those as we will have those soon